web/src/components/dsprot/DsProtMain.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+    <p>Double-spending Protection Service</p>
+    {{helloMessage}}
+    <TransactionList :transactions="transactions"/>
+</template>
+
+<script>
+    import TransactionList from "./TransactionList.vue"
+
+    export default {
+        name: 'DsProtMain',
+        props: {
+
+        },
+        components: {
+            TransactionList
+        },
+        data() {
+            return {
+                /*
+                * Declares empty transaction array, is filled at component creation time.
+                * Invariant: contains all transactions that the system is currently aware of.
+                */
+                transactions: [],
+                
+                // hard-coded message for heartbeat check of double-spending protection service
+                helloMessage: ''
```

Comment:
The comment confuses me, why is it hard-coded? Looks empty to me

Reply:
message is hard-coded in the backend, comment should be clearer in next commit